src/aioswitcher/api/__init__.py

```diff
@@ -652,6 +652,13 @@ def __init__(self, ir_set: dict) -> None:
             }
         }
         """
+        self._separated_swing_command = self._remote_id in [
```

Comment:
can you explain this list of specific devices IDs, please?

Reply:
The developer from Switcher.
These are the only Air Conditioner remote IDs that have a special handling with the way you control their Swing capability

Reply:
please set this list as a "constant" and comment better about what it does and the actual ac types it refers to.